sensorflux/database_connector.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+sensorflux.database_connector
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This module contains the class that communicates data to influxdb.
+"""
+
+from copy import deepcopy
+from datetime import datetime
+
+from influxdb import InfluxDBClient
+
+
+class DatabaseConnector:
+    """
+    A class to represent a client connection to the influxdb instance.
+
+    :param str measurement: measurement name.
+    :param str device: name for the device tag.
+    :param tuple fields: names of the fields containing the data to be sent.
+    """
+    def __init__(self, measurement, device, fields):
+        self.client = InfluxDBClient(
+            host='localhost',
+            port=8086,
+            username='admin',
+            password='admin',
+            database='sensorflux')
+        self.measurement = measurement
+        self.device = device
+        self.fields = fields
+        self._create_database('sensorflux')
+
+    def _create_database(self, db_name):
```

Comment:
I'd use a "ensure_database" (or similar), so you're more pertinent with what you're doing, since you're checking for its existence beforehand